README.md

```diff
@@ -1,2 +1,146 @@
-# exadel_discounts_be
-MVP
+# exadel_discounts_be

```

Comment:
File should be in markdown format - not html

Reply:
Fixed